sources/vtt_data/CascadiaCode.ttx

```diff
@@ -8708,7 +8708,7 @@ IUP[X]
 /* VTTTalk Unicode 0x2202 */
 /* ACT generated Mon Sep 14 09:57:28 2020 */
 
-SOFFSET[R], 624, 1202, 0, -1, 0, 0, 1
+OFFSET[R], 624, 1202, 0
```

Comment:
What's the cause of this change?

Reply:
vttLib does not know what SOFFSET is (I don't either... scaled offset?) but it seems useless so I removed the superfluous bits.

Reply:
Ah, you know, it looks like I had used a mirrored component there. So it is an offset with a transform applied. 
I should get rid of that in the source and add some hints. Will do that (later).

Reply:
Oooh wait, does this flip something? :open_mouth:

Reply:
So is this line valid still?

Reply:
@DHowett I don't have the time right now to make the fix. Will do so tomorrow morning. 
Because the glyph will change from a component one to an outline one, all of that will change.

Reply:
Sorry, just so we're on the same page: I should not ship a font built out of this script because this would damage one of the glyphs?
Sorry if that's a dense question :smile:

Reply:
Potentially. Never understood what VTT needs these OFFSET things for. I'll have a quick look with FreeType.

Reply:
I think you can actually just remove the OFFSET entirely and then ship it. That way it isn't being modified at all by hinting instructions and I can correct it when I have a chance. But I'd suggest checking 0x2022.

Reply:
So that particular line belongs to `partialdiff`, glyph ID 859 in CascadiaCode.ttf. FreeType shows no immediately visible diff between old and new build...

Reply:
(I must now sleep. One can always re-release another day!)

Reply:
I decided to hold this even though we're timing for a Terminal release -- no point in rushing it, and Terminal only has to work on Windows whereas the general-use build of this font has to work right everywhere :smile:

Reply:
Thanks @DHowett. Turns out I'll be able to get to this tonight, if that helps since we have such a blazing fast build process now ;). Or just wait until tomorrow because I'm sure you have better things to do. Haha.

Reply:
Eh, I already pressed the Terminal builds and it'll be nicer for us to release "2009.22" on Sep 22.

If you want to propose a change tot his file in another branch and @madig can merge master, I'm all for it.

Thanks :D